src/pingall.coffee

```diff
@@ -19,3 +22,25 @@ module.exports = (robot) ->
         message = context.message.text
         message = message.replace robot.name + ' pingall', ""
         context.send '[' + usersStr + ']' + " " + message
+
+
+module.exports = (robot) ->
+    robot.respond /add pingall cron (.*) "(.*?)" *$/i, (context) ->
+        room = context.message.room
+
+        users = Object.keys context.robot.adapter.bot.chans[room].users
+        usersStr = users.toString()
+
+        # format cronjob
+        timezone = 'America/Montreal'
+        pattern = context.match[1]
+        message = context.match[2]
+
+        pingallCron = () ->
+            context.send '[' + usersStr + ']' + " " + message
+        
+        try
+            new HubotCron pattern, timezone, pingallCron
```

Comment:
Does hubot sends the cron messages only on the chan where he received the command ?